modules/compute/gke-node-pool/gpu-direct-workload/scripts/enable-tcpx-in-workload.py

```diff
@@ -0,0 +1,182 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import yaml
+import argparse
+
+def main():
+    parser = argparse.ArgumentParser(description="TCPX Job Manifest Generator")
+    parser.add_argument("-f", "--file", required=True, help="Path to your job template YAML file")
+    parser.add_argument("-r", "--rxdm", required=True, help="RxDM version")
```

Comment:
nit: how can the customer find this version ? Can we help out here ?

Reply:
I have checked with the NetInfra team and the GKE AI Infra team about this, unfortunatly there is no plan for publicly sharing this information. The best we get is the google doc with restricted access [a3-tcpx](https://docs.google.com/document/d/1bHQLTMcP7IUm0xLB-rD2wKsZa6z6Uy6uD1W_RzqW_kw/edit?tab=t.0#heading=h.ifagugkyp9ak) and [a3mega-tcpxo](https://docs.google.com/document/d/1D5umT4-WDuNnYf3ieQ5SfLdmvGRPBQGLB662udzwz8I/edit?tab=t.0#heading=h.n4ytmbxt737h)

Reply:
Going back to nick's comment on storing the mapping between `rxdm` version and `nccl-test` installer. Can we store and check validated `rxdm` versions against the internally known good versions ?

Reply:
discussed offline, since we are no longer taking input from user for the rxdm/nccl version, I think we probably better off just provide a fixed combination for user

added warning in the output instruction about the version mapping